### PR TITLE
Added new units and aliases - updated units.json

### PR DESCRIPTION
**Pull request by sood-akash823**
New units added - Mbpd and cf/bbl
new alias added for cubic foot, cubic foot per day and kilo ft3

### CODE REVIEW

**Comment by sood-akash823** [comment]
Hi Team,

Can you please help me why the build failed?

Thanks,
Akash

File: versions/v1/units.json [hunk 1]
```diff
@@ -10590,5 +10598,50 @@
...
+    "conversion": {
+      "multiplier": 0.0018401307283,
+      "offset": 0.0
+    },
+    "source": "Energistics",
+    "sourceReference": "https://www.energistics.org/energistics-unit-of-measure-standard"
+  },
+  {
+    "externalId": "volume_fraction:ft3-per-bbl_us",
+    "name": FT3-PER-BBL_US",
```
Comment (polomani) [inline versions/v1/units.json:10628]:
```suggestion
    "name": "FT3-PER-BBL_US",
```
json is invalid

File: versions/v1/units.json [hunk 2]
```diff
@@ -10590,5 +10598,50 @@
...
+	  "kilo barrels per day",
+	  "thousand barrels per day",
+      "kbpd",
+	  "kb/d",
+	  "Mb/d",
+	  "MBPD",
+	  "Mbpd",
+	  "Mbbl/d",
+	  "kBbl/day"
+    ],
```
Comment (polomani) [inline versions/v1/units.json:10617]: need to run a formatter, here and similar situation below

Comment (sood-akash823) [reply versions/v1/units.json]: @polomani ran the entire json in a json formatter. Please verify

Comment (polomani) [reply versions/v1/units.json]: this is too much, can you make sure the format you use only formats the fields you changed?
if not, please format manually what you changed, so we can keep proper git history.

Comment (sood-akash823) [reply versions/v1/units.json]: let me raise a new PR for this. I will close this PR from my end

**Comment by sood-akash823** [comment]
Closing this PR. Raising a new one for same changes with correct formatting

**Pull request closed without merging**